cdragontoolbox/hashes.py

```diff
@@ -328,6 +328,39 @@ def substitute_basename_words(self):
     def add_basename_word(self):
         super()._add_basename_word(self.build_wordlist())
 
+    def multi_substitution(self, plugin, fileext=""):
+        """Replaces a word in all basenames while adding an additional word near it.
+        Because of high running time, a specific plugin must and a file extension can be specified"""
```

Comment:
Words are missing after `must`

Reply:
This is meant as in "a specific plugin must be specified and a file extension can be specified", just in shorter.